01/dylannakahodo/wordvalue.py

```diff
@@ -0,0 +1,33 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    """Load dictionary into a list and return list"""
+    with open(DICTIONARY) as dictionary:
+        content = dictionary.readlines()
+    return [line.strip() for line in content]
+
+def calc_word_value(word):
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+    value = 0
+    for char in word:
+        try:
+            value += LETTER_SCORES[char.upper()]
+        except:
+            pass
```

Comment:
"Explicit is better than implicit" - better to use a specific Exception on except (not except/pass), see https://pybit.es/py-mistakes.html

Reply:
Thanks for reviewing! I realize this is a poor/lazy way of handling the key lookup in the dict after seeing other people's solutions.

Reply:
right: use dict.get which defaults to None or a default value you give it. 

I (Bob) ended up with this one-liner for calc_word_value:     

return sum(LETTER_SCORES.get(char.upper(), 0) for char in word)

Reply:
you can do this one here too now: https://codechalleng.es/bites/3/

Reply:
Cool thanks! I'll check it out.